src/component/renderable/schema/part-control/auto-fill/assets/popup-layout.html

```diff
@@ -1,13 +1,11 @@
 <div data-bc-autocomplete-popup-container>
-    <div data-bc-autocomplete-popup="" data-sys-autocomplete-popup="">
-        <div data-bc-title="" data-sys-autocomplete-popup-header="">
-            <button data-bc-btn-close="" data-sys-text="">X</button>
-        </div>
-        <div data-bc-body="" data-sys-autocomplete-popup-body="">
-            <input data-bc-search type="text"  data-sys-input-text="" />
-            <ul data-bc-result="" data-sys-search-result="">
-    
-            </ul>
-        </div>
+  <div data-bc-autocomplete-popup="" data-sys-autocomplete-popup="">
+    <div data-bc-title="" data-sys-autocomplete-popup-header="">
+      <button data-bc-btn-close="" data-sys-text="">X</button>
     </div>
-</div>
\ No newline at end of file
+    <div data-bc-body="" data-sys-autocomplete-popup-body="">
+      <input data-bc-search type="text" data-sys-input-text="" />
+      <ul data-bc-result="" data-sys-search-result=""></ul>
+    </div>
+  </div>
+</div>
```

Comment:
Replace all `popup` with `html` in contents and file names (if popup renamed to html)